picturepyfect/rotate_pyfect.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+
+
+def rotate_pyfect(image, n_rot=90):
```

Comment:
I think you've switched to number of rotations instead of degrees correct? If that's the case I sugges
```suggestion
def rotate_pyfect(image, n_rot=1):
```